src/symbol/observable.ts

```diff
@@ -0,0 +1,18 @@
+import {root} from '../util/root';
+
+const Symbol: any = root.Symbol;
+
+export let $$observable: any;
+
+if (typeof Symbol === 'function') {
+  if (!Symbol.observable) {
+    if (typeof Symbol.for === 'function') {
+      $$observable = Symbol.for('observable');
+    } else {
+      $$observable = Symbol('observable');
```

Comment:
@trxcllnt Line 14 below patches it onto the global Symbol.

@blesh Let's say you have frame A and frame B.  RxJS gets loaded into both frames separately.  Since the `Symbol` function itself is not shared between frames (each frame has its own copy) both RxJS instances will polyfill `Symbol.observable` with a different symbol value.  If you try to give an Observable from frame B to frame A, the RxJS in frame A won't recognize the symbol that was polyfilled into frame B, and therefore it won't recognize it as an "observable-able".

`Symbol.for` doesn't have that problem because it uses a cross-realm global registry of string names to symbol values.  Because of the cross-frame issues, I'd only polyfill if `Symbol.for` was available.  But in that case, there's not much point in polyfilling.  All libraries can just use `Symbol.for("observable")` if `Symbol.observable` doesn't exist.  And of course fall back to `"@@observable"` if `Symbol.for` is not available.